auth/test/TestProviderMT51Scheduler.m

```diff
@@ -7,14 +7,14 @@ function test_scheduler_with_two_keystacks(testcase)
             current_hash_path_end = uint8(randi([0, 255], 16, 1));
             next_hash_path_end = uint8(randi([0, 255], 16, 1));
 
-            mt51_scheduler = MT51Scheduler(current_time, current_hash_path_end, next_hash_path_end);
+            mt51_scheduler = ProviderMT51Scheduler(current_time, current_hash_path_end, next_hash_path_end);
```

Comment:
I got a similiar error on every test that 

Undefined function 'SBASKeyStack' for input arguments of type 'uint32'.

let me know how to fix it so I can step though this test file as well as the other that I missed please